lib/reverse_sentence.rb

```diff
@@ -1,6 +1,59 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n*m) (where n is the length of the string and m is the length of the longest word in the string)
+# Space complexity: O(1) (the same number of variables are created at the same length regardless of input)
+
 def reverse_sentence(my_sentence)
```

Comment:
👍 This works, but notice that you're reversing the whole string and then reversing each word, could you create a helper method and then re-use it in each case?